examples/examples/ws_pubsub_with_params.rs

```diff
@@ -67,9 +67,10 @@ async fn run_server() -> anyhow::Result<SocketAddr> {
 	let mut module = RpcModule::new(());
 	module
 		.register_subscription("sub_one_param", "sub_one_param", "unsub_one_param", |params, pending, _| {
-			let (idx, mut sink) = match (params.one(), pending.accept()) {
-				(Ok(idx), Some(sink)) => (idx, sink),
-				_ => return,
+			let idx = params.one()?;
+			let mut sink = match pending.accept() {
+				Some(sink) => sink,
+				_ => return Ok(()),
```

Comment:
better to change `fn accept() - Result<SubscriptionSink, Error` this `return Ok(())` is confusing for most folks.

I get it because at this point jsonrpsee has no use of the error because the connection is closed.